package/MDAnalysis/coordinates/PDB.py

```diff
@@ -389,9 +389,13 @@ def _read_frame(self, frame):
                 pos += 1
             elif line[:6] == 'CRYST1':
                 # does an implicit str -> float conversion
-                self.ts._unitcell[:] = [line[6:15], line[15:24],
-                                        line[24:33], line[33:40],
-                                        line[40:47], line[47:54]]
+                try:
+                    self.ts._unitcell[:] = [line[6:15], line[15:24],
+                                            line[24:33], line[33:40],
+                                            line[40:47], line[47:54]]
+                except ValueError:
+                    warnings.warn("Failed to read CRYST entry, got:\n{}"
```

Comment:
Maybe be even more explicit and say "Failed to read CRYST1 record, possibly invalid PDB file: read:\n{}"

Reply:
Just to make sure that user's understand that this is a problem with the file and not with MDAnalysis.